packages/astro/test/fixtures/custom-page-options/src/pages/index.astro

```diff
@@ -0,0 +1,15 @@
+---
+export const prerender = true;
+
+export const customTrue = true;
+export const customFalse = false;
```

Comment:
We should add more cases:
- no semicolons
- more spaces between `=` and the value;